.github/workflows/common.playwright.yml

```diff
@@ -9,15 +9,21 @@ on:
     secrets:
       READER_TOKEN:
         required: true
+
+permissions:
```

Comment:
Sikker på at disse trengs her? Settes jo i caller, dvs. app.dolly-frontend.yml og hvis du ser på hvordan all.workflows.yml henger sammen med common.frontend.yml f.eks. så settes permissions globally fra all.workflows.yml her. Prøv gjerne å flytte permissions.* i app.dolly-frontent.yml opp et nivå (ref. all.workflows.yml) og se om det funker uten øvrige referanser til permissions i common.frontend.yml. Grunnen til at det ikke settes globally i app.*.yml er at det er én job som kjøres, hvis jeg husker egen tankegang rett. Kunne vært kjekt å teste da. Men vil funke as is. :)